api/src/pcapi/alembic/versions/20250227T165242_b587bd63c5b6_add_future_offers_reminders_constraints.py

```diff
@@ -0,0 +1,36 @@
+"""Add FutureOffer Reminders constraints"""
+
+from alembic import op
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "b587bd63c5b6"
+down_revision = "c34443975234"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    op.create_foreign_key(
+        "future_offer_reminder_userId_fkey",
+        "future_offer_reminder",
+        "user",
+        ["userId"],
+        ["id"],
+        ondelete="CASCADE",
+        postgresql_not_valid=True,
+    )
+    op.create_foreign_key(
+        "future_offer_reminder_futureOfferId_fkey",
+        "future_offer_reminder",
+        "future_offer",
+        ["futureOfferId"],
+        ["id"],
+        ondelete="CASCADE",
+        postgresql_not_valid=True,
+    )
+
+
+def downgrade() -> None:
+    pass
```

Comment:
est-ce qu'on veut pas faire ça dans le downgrade : 

```suggestion
op.drop_constraint("future_offer_reminder_userId_fkey", "future_offer_reminder", type_="foreignkey")
op.drop_constraint("future_offer_reminder_futureOfferId_fkey", "future_offer_reminder", type_="foreignkey")
```
?

Reply:
je ne l'ai pas vu dans les autres parties du code qui font l'ajout des contraintes, mais je vais double check ça !